Populate UpperCase validator whitelist with NSI data

malteger opened the pull request:
This adds support for NSI data also to the UpperCase plugin. The whitelist from NSI data and the whitelist as defined in the plugin are merged and used in the validator.

Follow up to #1607 

Fixes #1428

Famlam commented:
#1428  
Are the "hardcoded" whitelists still needed?

frodrigo commented:
> #1428 Are the "hardcoded" whitelists still needed?

FR and ES list are not brand part of NSI.

frodrigo commented:
Adding NSI the brand list should be rewrite like this
```
    "FR": ["CNFPT", "COSEC", "EHPAD",  "MEDEF", "URSSAF"],
```

frodrigo commented:
@malteger Thank you to do this. Can you change the whitelists content, please.

malteger commented:
I changed the french whitelist according to your suggestion and removed the german one again, as it only contains brand names which are covered by NSI.

I think it is good to keep the hardcoded whitelist for things which are not brands.

Famlam commented:
> I think it is good to keep the hardcoded whitelist for things which are not brands.

Agree, apologies for being unclear, I indeed only meant removing the "now dealt-with via NSI" items

frodrigo commented on plugins/Name_UpperCase.py:
```diff
@@ -22,12 +22,13 @@
 UpperCase_WhiteList = {
-    "FR": ["CNFPT", "COSEC", "EHPAD", "MACIF", "MEDEF", "URSSAF"],
+    "FR": ["CNFPT", "COSEC", "EHPAD", "MEDEF", "URSSAF"],
     "ES": ["FREMAP"],
```
Spanish is also a brand.

```suggestion
```

frodrigo commented:
Merged, but not deployed yet. Thank you.
